sim-lib/src/lib.rs

```diff
@@ -73,3 +114,30 @@ struct PaymentResult {
     settled: bool,
     action: u8,
 }
+
+pub struct Simulation {
+    // The lightning node that is being simulated.
+    nodes: HashMap<String, Arc<dyn LightningNode>>,
```

Comment:
Comments are for single-node but structs are for multi node, would just update comments.